<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimalProfilWindow</class>
 <widget class="QDialog" name="AnimalProfilWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelImg">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>181</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Navn</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_age">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>alder</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_owner">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Eier</string>
   </property>
  </widget>
  <widget class="QLabel" name="labe_tlf">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tlf</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Animal:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Owner:</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_log">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>471</width>
     <height>351</height>
    </rect>
   </property>
   <property name="autoScrollMargin">
    <number>0</number>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>180</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnLogFood">
      <property name="text">
       <string>Fed</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnLogClean">
      <property name="text">
       <string>Cleaned cage</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnLogCustom">
      <property name="text">
       <string>Custom</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>112</width>
     <height>49</height>
    </rect>
   </property>
   <property name="text">
    <string>Logging functions: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sex">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>label_sex</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_specialNeeds">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>sn</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
